src/lib/rest/StringFilter.cpp

```diff
@@ -0,0 +1,1101 @@
+/*
+*
+* Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Yes, that is true.
With all the problems I had, I forgot to remove the old stuff